backend/gis_module.py

```diff
@@ -0,0 +1,47 @@
+# Module for common GIS utilities relevant to shoulder2shoulder.
+from geopy.geocoders import Nominatim
+from geopy import distance
+
+def geocode(address):
```

Comment:
When I try to run this from within the backend, I get an SSL Verification error. I was able to figure out a workaround:

```python
import certifi
import ssl
import geopy.geocoders
from geopy.geocoders import Nominatim

# set up certificates
ctx = ssl.create_default_context(cafile=certifi.where())
geopy.geocoders.options.default_ssl_context = ctx

def geocode(address):
    '''
    Geocode an address as a string and returns dictionarity with the standardized address and a 
    returns a 'coords' key with a tuple that has latitude, then longitude.
    '''

    # Nominatim has a rate limit of 1 second. For testing purposes, 
    # I am using my (Ethan's) email as the registered user.
    geolocator = Nominatim(user_agent = "ethanarsht@gmail.com", scheme='http')
    location = geolocator.geocode(address)

    return {'address': location.address, 'coords': (location.latitude, location.longitude)}
```